rmw/include/rmw/get_topic_endpoint_info.h

```diff
@@ -31,6 +31,15 @@ extern "C"
  * the associated topic type, the publisher's gid, and the publisher QoS profile.
  * Names of non-existent topics are allowed, in which case an empty array will be returned.
  *
+ * \par QoS that are correctly read
+ *   The QoS profiles returned might have some invalid fields.
+ *   Particularly, the only QoS policies that are guaranteed to be shared
+ *   during discovery are the ones that participate in endpoint matching.
+ *   From the current QoS settings available, the only ones not shared by DDS based
+ *   implementations are `history` and `history_depth`.
+ *   The rmw implementation must set the settings that were not correctly read to
+ *   `RMW_QOS_POLICY_*_UNKNOWN`.
```

Comment:
I don't think this needs to be indented.

Based on this:

https://www.star.bnl.gov/public/comp/sofi/doxygen/commands.html#cmdpar

Any you just want `\par` for the "heading" right?

```suggestion
 *  The QoS profiles returned might have some invalid fields.
 *  Particularly, the only QoS policies that are guaranteed to be shared
 *  during discovery are the ones that participate in endpoint matching.
 *  From the current QoS settings available, the only ones not shared by DDS based
 *  implementations are `history` and `history_depth`.
 *  The rmw implementation must set the settings that were not correctly read to
 *  `RMW_QOS_POLICY_*_UNKNOWN`.
```

Reply:
I also wonder if the `section`/`subsection`/`subsubsection` commands are a better fit?

https://www.star.bnl.gov/public/comp/sofi/doxygen/commands.html#cmdsection

Reply:
I copied the style from another part of the same docblock https://github.com/ros2/rmw/blob/aa3a366a59d525ce54e1554b39f4721336601344/rmw/include/rmw/get_topic_endpoint_info.h#L52-L69

\section and \subsection seem to only work when combined with the \page command:

> Warning:
>   This command only works inside related page documentation and not in other documentation blocks!

I agree that the indentation is unnecesary though.